### PR TITLE
dejavu-ttf: new package at 2.37

### PR DESCRIPTION
**Pull request by eudaldgr**
## Description of package

The [DejaVu](https://dejavu-fonts.github.io/) fonts are a font family based on the Vera Fonts.

This package solves my issue with `cups-filters` #794 

## Installed manifest of package

```
/var/db/kiss/installed/dejavu-ttf/version
/var/db/kiss/installed/dejavu-ttf/sources
/var/db/kiss/installed/dejavu-ttf/post-install
/var/db/kiss/installed/dejavu-ttf/manifest
/var/db/kiss/installed/dejavu-ttf/checksums
/var/db/kiss/installed/dejavu-ttf/build
/var/db/kiss/installed/dejavu-ttf/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/fonts/TTF/DejaVuSerifCondensed.ttf
/usr/share/fonts/TTF/DejaVuSerifCondensed-Italic.ttf
/usr/share/fonts/TTF/DejaVuSerifCondensed-BoldItalic.ttf
/usr/share/fonts/TTF/DejaVuSerifCondensed-Bold.ttf
/usr/share/fonts/TTF/DejaVuSerif.ttf
/usr/share/fonts/TTF/DejaVuSerif-Italic.ttf
/usr/share/fonts/TTF/DejaVuSerif-BoldItalic.ttf
/usr/share/fonts/TTF/DejaVuSerif-Bold.ttf
/usr/share/fonts/TTF/DejaVuSansMono.ttf
/usr/share/fonts/TTF/DejaVuSansMono-Oblique.ttf
/usr/share/fonts/TTF/DejaVuSansMono-BoldOblique.ttf
/usr/share/fonts/TTF/DejaVuSansMono-Bold.ttf
/usr/share/fonts/TTF/DejaVuSansCondensed.ttf
/usr/share/fonts/TTF/DejaVuSansCondensed-Oblique.ttf
/usr/share/fonts/TTF/DejaVuSansCondensed-BoldOblique.ttf
/usr/share/fonts/TTF/DejaVuSansCondensed-Bold.ttf
/usr/share/fonts/TTF/DejaVuSans.ttf
/usr/share/fonts/TTF/DejaVuSans-Oblique.ttf
/usr/share/fonts/TTF/DejaVuSans-ExtraLight.ttf
/usr/share/fonts/TTF/DejaVuSans-BoldOblique.ttf
/usr/share/fonts/TTF/DejaVuSans-Bold.ttf
/usr/share/fonts/TTF/DejaVuMathTeXGyre.ttf
/usr/share/fonts/TTF/
/usr/share/fonts/
/usr/share/
/usr/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

**Comment by dylanaraps** [comment]
Could this same issue not be solved by installing **any** font? There are a bunch in the repositories.

```sh
/home/dylan/projects/repo/extra/liberation-fonts
/home/dylan/projects/community/community/font-awesome-otf
/home/dylan/projects/community/community/font-awesome-ttf
/home/dylan/projects/community/community/iosevka-nerd-fonts
/home/dylan/projects/community/community/terminus-font
/home/dylan/projects/community/community/unifont
```

**Comment by eudaldgr** [comment]
I have installed this fonts, and the problem persists, until i install `dejavu`
```
/var/db/kiss/installed/font-awesome-ttf
/var/db/kiss/installed/font-awesome-otf
/var/db/kiss/installed/freefont-ttf
/var/db/kiss/installed/liberation-fonts
/var/db/kiss/installed/terminus-font
```
I don't know about `iosevka-nerd-fonts` and `unifont`

File: community/dejavu-ttf/build [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+install -d "$1/usr/share/fonts/TTF"
```
Comment (dylanaraps) [inline community/dejavu-ttf/build:3]: Remove this line and change the next line to `-Dm644`.

Comment (eudaldgr) [reply community/dejavu-ttf/build]: Copy pasta

**Comment by eudaldgr** [comment]
If `cups-filters` needs some font, maybe it will be specified on depends file.

**Comment by dylanaraps** [comment]
I find it rather odd that a font is needed and I'm against making a package depend on a specific font to be installed... Anyway. I'll merge this PR for now.

**Pull request merged**